source/scpd/Cpp.d

```diff
@@ -76,40 +79,39 @@ private nothrow @nogc extern(C++) void cpp_unordered_map_assign (K, V)(
 private pure nothrow @nogc @safe extern(C++) size_t cpp_unordered_map_length (K, V)(
     const(void)* map);
 
-// @bug with substitution
-// https://issues.dlang.org/show_bug.cgi?id=20679
-//private pure nothrow @nogc @safe extern(C++) unordered_map!(K, V)* cpp_unordered_map_create (K, V)();
-
-/// Create a new unordered map with the given Key/T types, and return a pointer to it
-private pure nothrow @nogc @safe extern(C++) void* cpp_unordered_map_create (K, V)();
-
 /// Rudimentary bindings for std::unordered_map
 extern(C++, (StdNamespace))
 public struct unordered_map (Key, T, Hash = hash!Key, KeyEqual = equal_to!Key, Allocator = allocator!(pair!(const Key, T)))
 {
-    private void* ptr;
+    private ulong[56 / ulong.sizeof] _data;
```

Comment:
`ulong.sizeof` is *always* 8. Did you mean `size_t` or `cpp_ulong` ?

Reply:
Nah, I made that a `ulong[]` hoping that it would help with alignment and hard coding length to 7 just looked odd.